### PR TITLE
Clean up samples

### PR DESCRIPTION
**Pull request by sebbert**
The majority of these changes are:
- Slight code style fixes
- `Observable.map().inner()` => `Observable.flatMap()`
- Remove explicit ux:Class on `<App>`
- Change `<App ClearColor` to `<App Background`

### CODE REVIEW

File: Samples/FilterOnObservableCondition/MainView.js [hunk 1]
```diff
@@ -35,11 +35,11 @@ var items = Observable(
 	new Color('#FFFF00', 'Other 5', COLOR_TYPES.OTHER)
 );
 
-var filteredItems = currentCategory.map(function(category){
```
Comment (yupferris) [inline Samples/FilterOnObservableCondition/MainView.js:38]: This example is also shown here: https://www.fusetools.com/docs/fusejs/observable#where-condition

Perhaps it's correct _not_ to use `flatMap` here, or? I'm actually not sure

File: Samples/ForeignAccelerometer/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# ForeignAccelerometer
```
Comment (yupferris) [inline Samples/ForeignAccelerometer/README.md:1]: Did this branch add the ForeignAccelerometer exmaple, or was that on a branch or something?

**Review by yupferris — CHANGES_REQUESTED** [review]
Looks good, just need the review addressed before merging :)

File: Samples/ForeignAccelerometer/README.md (hunk 2 above)
Comment (sebbert) [reply Samples/ForeignAccelerometer/README.md:1]: It was already there, but I forgot to make a README and add it to the table of contents when I wrote that example.

File: Samples/FilterOnObservableCondition/MainView.js (hunk 1 above)
Comment (sebbert) [reply Samples/FilterOnObservableCondition/MainView.js:38]: Then the Observable docs are outdated.

For single-value Observables, `map(fn).inner()` is equivalent `flatMap(fn)`.
I've also verified that the example behaves the same.

Comment (yupferris) [reply Samples/FilterOnObservableCondition/MainView.js:38]: including the propagation/update details? Just want to make sure we're certain

Comment (sebbert) [reply Samples/FilterOnObservableCondition/MainView.js:38]: See for yourself ;)

https://github.com/fusetools/fuselibs/blob/db8c54a4f663670812a02e2f9a90d9a3344a1a65/Source/Fuse.Reactive/FuseJS/Observable.js#L754

Comment (yupferris) [reply Samples/FilterOnObservableCondition/MainView.js:38]: hoookay lol

**Review by yupferris — APPROVED** [review]

**Pull request merged**